frontend/templates/post-edit.tera.html

```diff
@@ -1,317 +0,0 @@
-{% import "page.tera.html" as page %}
```

Comment:
This file along with `post-list.tera.html` and `post-send.tera.html` are rebase artifacts and should be removed